mal/useranimelist_test.go

```diff
@@ -134,6 +136,8 @@ func TestAnimeServiceUpdateMyListStatus(t *testing.T) {
 		RewatchValue(1),
 		Tags{"foo", "bar"},
 		Comments("comments"),
+		StartDate(time.Date(2022, 02, 20, 0, 0, 0, 0, time.UTC)),
+		FinishDate(time.Date(2022, 04, 16, 0, 0, 0, 0, time.UTC)),
```

Comment:
We can cover adding and removing dates in the same test e.g. add a start date and remove a finish date.
```suggestion
		FinishDate(time.Time{}),
```